data/games/guilty_gear_xrd_rev_2/movespecs/potemkin.json

```diff
@@ -0,0 +1,125 @@
+{
+  "name": "Potemkin",
+  "moves": {
+    "normals": [
+      "5P",
+      "5K",
+      "c.S",
+      "f.S",
+      "5H",
+      "5D",
+      "6P",
+      "6K",
+      "6H",
+      "2P",
+      "2K",
+      "2S",
+      "2H",
+      "2D",
+      "j.P",
+      "j.K",
+      "j.S",
+      "j.H",
+      "j.D"
+    ],
+    "specials": [
+      {
+        "name": "Heat Knuckle",
+        "inputs": [
+          "623H"
+        ],
+        "follow_ups": [
+          {
+            "name": "Heat Extend",
+            "inputs": [
+              "63214H"
+            ]
+          }
+        ]
+      },
+      {
+        "name": "Megafist",
+        "inputs": [
+          "236P",
+          "214P"
+        ]
+      },
+      {
+        "name": "Slide Head",
+        "inputs": [
+          "236S"
+        ]
+      },
+      {
+        "name": "Hammer Fall",
+        "inputs": [
+          "[4]6H"
+        ],
+        "follow_ups": [
+          {
+            "name": "Brake",
+            "inputs": [
+              "P"
+            ],
+            "conditions": "Hammer Fall hasn't hit"
+          }
+        ]
+      },
+      {
+        "name": "F.D.B.",
+        "inputs": [
+          "63214S"
```

Comment:
@yithian is this correct? i was unable to find anything on dustloop about a held `S` variant for this move.

Reply:
Yeah, Pot can hold S for a bit to delay the flick. It's listed as a bullet point in F.D.B.

Reply:
Ah okay, cool. thank you

Reply:
dangit, i was looking at the revalator moveset, not rev2. :anguished: